index.html

```diff
@@ -107,6 +107,74 @@ <h3>
       </dd>
     </dl>
   </section>
+  <section id="RTCVideoReceiverStats-dict*">
+    <h3>
+      <dfn>RTCVideoReceiverStats</dfn> dictionary non-standardized members
+    </h3>
+    <pre class="idl">partial dictionary RTCVideoReceiverStats {
+      unsigned long numFreezes;
+      unsigned long numPauses;
+      double totalFreezesDuration;
+      double totalFramesDuration;
+      double sumOfSquaredFramesDuration;
+};</pre>
+    <dl data-link-for="RTCVideoReceiverStats" data-dfn-for="RTCVideoReceiverStats" class=
+    "dictionary-members">
+      <dt>
+        <dfn><code>numFreezes</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long">unsigned long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video freezes experienced by this receiver. It is a
+          freeze if frame duration, which is time interval between consecutively
+          rendered frames, is 3 times higher than the average frame duration.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>numPauses</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total number of video pauses experienced by this receiver. Video is
+          considered to be paused if time passed since last received packet
+          exceeds 5 seconds.
+        </p>
+      </dd>
+      <dt>
+        <dfn><code>totalFreezesDuration</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-double">double</a></span>
+      </dt>
+      <dd>
+        <p>
+          Total duration of video freezes, in seconds.
```

Comment:
How is this measured? If the average interval is 10 ms, so that "3 times higher than the average" is 30 ms, and it takes me 50 ms to receive a frame: does totalFreezesDuration increase by 20 ms or 50 ms?

When is this value calculated? Is it updated when getStats() is called or when a frame arrives? I.e. if I don't receive any frames, does this value increase with +1s/s despite not receiving any new frames?

Reply:
How about this definition?:
Total duration of rendered frames which are considered as frozen (see freezeCount for definition of freeze), in seconds. This value is updated when a frame is rendered.